incubator-tools/cdc_document_type_entity_addition/cdc_document_type_entity_addition.ipynb

```diff
@@ -0,0 +1,240 @@
+{
```

Comment:
<p>I don't think this notebook is needed anymore since it's now possible to import directly from folder labels in CDC <a href="https://cloud.google.com/document-ai/docs/workbench/build-custom-classification-processor#optional_batch_label_documents_at_import" target="_blank">https://cloud.google.com/document-ai/docs/workbench/build-custom-classification-processor#optional_batch_label_documents_at_import</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/document-ai-samples/pull/755/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Removed the tool.